### PR TITLE
DOCS-2831: Update machine cloud credentials terminology

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/configure/agent.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Your machine must have `curl` available in order to install `viam-agent`.
     {{< alert title="Note" color="note" >}}
 
 As an alternative to specifying the `VIAM_API_KEY_ID`, the `VIAM_API_KEY`, and the `VIAM_PART_ID` when running the command, you can also copy the `viam-server` app JSON configuration from the Viam app into <file>/etc/viam.json</file>.
-You can get the `viam-server` app JSON configuration by clicking the copy icon next to **Viam server config** in the part status dropdown to the right of your machine's name on the top of the page.
+You can get the `viam-server` app JSON configuration by clicking the copy icon next to **Machine cloud credentials** in the part status dropdown to the right of your machine's name on the top of the page.
```
Comment (JessamyT) [inline docs/configure/agent.md:56]:
```suggestion
You can get the `viam-server` cloud credentials by clicking the copy icon next to **Machine cloud credentials** in the part status dropdown to the right of your machine's name on the top of the page.
```
I think?

File: docs/fleet/provision.md [hunk 2]
```diff
@@ -102,16 +102,16 @@ This is the general process for provisioning depending on whether you are using
...
-     To copy a machine credentials configuration file:
+     To copy a machine cloud credentials file:
 
      - Navigate to your machine's page on [the Viam app](https://app.viam.com).
      - Select the part status dropdown to the right of your machine's name on the top of the page.
-       {{<imgproc src="configure/machine-part-info.png" resize="500x" declaredimensions=true alt="Restart button on the machine part info dropdown">}}
-     - Click the copy icon next to **Viam server config** to copy the machine credentials configuration file.
-     - Paste the `viam-server` app config when prompted.
+       {{<imgproc src="configure/machine-part-info.png" resize="500x" declaredimensions=true alt="machine cloud credentials button on the machine part info dropdown">}}
+     - Click the copy icon next to **Machine cloud credentials** to copy the machine cloud credentials file.
```
Comment (JessamyT) [inline docs/fleet/provision.md:113]:
```suggestion
     - Click the copy icon next to **Machine cloud credentials**.
```
Can we be less verbose? I think this is self-explanatory

**Review by JessamyT — APPROVED** [review]
LGTM % 2 small comments

File: docs/configure/agent.md (hunk 1 above)
Comment (npentrel) [reply docs/configure/agent.md:56]:
```suggestion
You can get the machine cloud credentials by clicking the copy icon next to **Machine cloud credentials** in the part status dropdown to the right of your machine's name on the top of the page.
```

**Pull request merged**